src/smtsolvers/CoreSMTSolver.cc

```diff
@@ -2112,7 +2112,7 @@ bool CoreSMTSolver::scatterLevel()
 }
 
 
-bool CoreSMTSolver::createSplit_scatter(bool last)
+bool CoreSMTSolver::createSplit_scatter(bool)
```

Comment:
@aehyvari , and this is the second example where the parameter should be probably removed completely?!

Reply:
It looks like this can be removed.  It's not used in parallel-infra either, and there it should be tested at least.